modules/grandpa/src/lib.rs

```diff
@@ -270,7 +275,7 @@ pub mod pallet {
 	/// Hash of the best finalized header.
 	#[pallet::storage]
 	pub type BestFinalized<T: Config<I>, I: 'static = ()> =
-		StorageValue<_, BridgedBlockHash<T, I>, ValueQuery>;
+		StorageValue<_, (BridgedBlockNumber<T, I>, BridgedBlockHash<T, I>), OptionQuery>;
```

Comment:
**Need migration**, Maybe we add a migration code to this pallet because these migration code are needed in most of our chains.